zenithdb/website
# feat & refactor: misc blog leftover tasks

**Describe changes**

This PR brings various tasks [from this Notion page](https://www.notion.so/pixelpoint/Blog-leftovers-bb1c6b6dc19e439f9c62243188c8eb5e#3504b1e5232b4644bf08b2218e117723)

**Steps to test**
1. Go to [_/privacy-policy_](https://clever-wescoff-fd0120.netlify.app/privacy-policy) page as an example of a static page in this project
2. Confirm everything works and looks as expected

NB! Please review the code locally, as deploy preview is unavailable for now. 

**TODO**

- [x] Find a way to make embedded twitter card make responsive 
- [ ] Delete all posts before merging PR and leave only one as an example without much content
- [ ] Respond to additional designers and devs comments, if there are any
